### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by johnkeough**
Screen Shot 2020-09-22 at 4 44 06 PM
Screen Shot 2020-09-22 at 4 44 18 PM
Screen Shot 2020-09-22 at 4 44 24 PM
Screen Shot 2020-09-22 at 4 44 36 PM
Screen Shot 2020-09-22 at 4 44 47 PM
Screen Shot 2020-09-22 at 4 44 55 PM
Screen Shot 2020-09-22 at 4 45 15 PM
Screen Shot 2020-09-22 at 4 45 29 PM
Screen Shot 2020-09-22 at 4 45 41 PM
Screen Shot 2020-09-22 at 4 45 55 PM

### CODE REVIEW

**Comment by gchtr** [comment]
Hey @johnkeough 

I see you’re still working on the styles. I only have a heads up: It would be great if you could exclude all the files that are built automatically from your pull requests, like all the files in the **build** or **docs** folder. It’s easier to look at the changes without these files.

Tell me if you need help doing that.

Looking forward to see your work! :)

**Comment by johnkeough** [comment]
Hey @gchtr !

Absolutely. I haven't before though, would you mind giving me a few pointers on how to do that?

Thank you! :)

**Comment by gchtr** [comment]
Okay, let’s see. What do you use to manage Git? An app like Sourcetree, Fork or GitHub Desktop or do you use Git with the CLI?

**Comment by johnkeough** [comment]
I use Git with the CLI for mostly all, no external apps.

File: assets/sass/_headings.scss [hunk 1]
```diff
@@ -32,7 +31,8 @@ h2 {
         width: 100%;
         margin-top: 0.25em;
         height: 2px;
-        background: linear-gradient(to right, $green-light, rgba(0, 0, 0, 0) 75%);
+        // background: linear-gradient(to right, $green-light, rgba(0, 0, 0, 0) 75%);
```
Comment (stephalee) [inline assets/sass/_headings.scss]: delete commented out code

File: assets/sass/_headings.scss [hunk 2]
```diff
@@ -53,8 +53,9 @@ h5 {
 }
 
 .heading-small {
-    font-size: 0.875rem;
-    text-transform: uppercase;
-    color: $meta;
-    letter-spacing: 1.5px;
+    // font-size: 0.875rem;
```
Comment (stephalee) [inline assets/sass/_headings.scss]: delete commented out code

File: assets/sass/_sidebar.scss [hunk 3]
```diff
@@ -37,16 +37,18 @@
...
 
 	a {
+		color: $white;
 		display: block;
-		border: 1px solid $green;
-		border-radius: 3px;
-		padding: 0 0.5em;
+		border: 1px solid $white;
+		border-radius: 100px;
+		// padding: 0 0.5em;
```
Comment (stephalee) [inline assets/sass/_sidebar.scss]: commented out code

File: assets/sass/_toc.scss [hunk 4]
```diff
@@ -1,6 +1,7 @@
 aside {
 	padding: 0 0 1.5em 0;
-	border-bottom: 1px solid $border-color;
+	margin-top: 3em;
+	// border-bottom: 1px solid $border-color;
```
Comment (stephalee) [inline assets/sass/_toc.scss]: commented out code

File: assets/sass/_toc.scss [hunk 5]
```diff
@@ -26,6 +28,10 @@ aside, .toc {
 
 	li {
 		display: block;
+		border-bottom: 1px solid #323436;
```
Comment (stephalee) [inline assets/sass/_toc.scss]: can we use a color var here?

**Review by stephalee — COMMENTED** [review]
Looking good @johnkeough! In general, I'd try to remove all commented out styles before merging, I work this way when trying out new colors too (: but once you feel good about the changes you'd made I'd just make sure all of those are removed. I commented on a few of them but didn't want to be repetitive. Still looking into what the best way to exclude the `docs` and `build` folder files is

**Comment by gchtr** [comment]
I think you can do

```
git checkout origin/master -- ./docs && git checkout origin/master -- ./content
```

to revert the changes to the built files (according to [this Stack Overflow answer](https://stackoverflow.com/a/44355701/1059980)). 

Also, did you make sure that you checked out the [`2.x`](https://github.com/timber/timber/tree/2.x) branch in the Timber repository that lies next to the docs repository? I realized there are some old file headers in the **content** folder. Sorry about that, I guess I didn’t document that in the Readme, because the docs are still in a transition state until the `2.x` branch is merged.

**Review by gchtr — CHANGES_REQUESTED** [review]
Wow, that looks great! I’ve always been in love with the GT America font. And I love what you did with the syntax higlighted parts! I’ve yet to get used to all the dark modes and interfaces. I saw that it’s based on the new [Upstatement](https://www.upstatement.com/) design. Nice!

I only have a couple of points where I think we have to find a compromise between design decisions and the reality that docs are a daily tool for developers that needs to readable and easy to navigate.

I saw that you only added light font styles. I guess now all strong text is rendered with a faux bold style, which is generated by the browser. Do you have a bold or even an extra bold font that you could add? I know, it’s a design decision to not use bold. But in the Timber documentation, we kinda use the convention to use `<strong>` to mark file names and file paths.

To me, the problem can also be seen in the navigation. Here, I think it’s too hard to see on which page I currently am:

grafik

Currently, there’s no way to see where there’s a link. Take this example:

grafik

There are actually two links in here:

grafik

It’s important to not only style text links, but also `<code>` elements that are links.

I realized it might not even be enough to style links with color only, as it’s done in the [current documentation](https://timber.github.io/docs/v2). We should probably add underlines to links in content, because we already use colored text a lot in the content when using `<code>` elements. That would also be better for acessibility, especially for people with a form of color blindness.

It’s a different thing with navigations or the table of contents, which consist mostly only of links. But then there are elements like tables with an overview over object properties and methods, where sometimes there are links, and sometimes there aren’t.

grafik

It would be great if you could improve on these points :).

**Comment by jarednova** [comment]
@gchtr thanks for the helpful feedback as always. @johnkeough got called away this week for another project, but when he returns I'll help him address these points in the next go!

**Comment by jarednova** [comment]
@johnkeough let's talk next week about a solution to the font issue. We can serve [crossdomain fonts](https://davidwalsh.name/cdn-fonts) from upstatement.com, but I'm not sure it's worth the effort. FWIW I think it looks glorious in Helvetica.

**Comment by johnkeough** [comment]
Thank you for the great feedback @gchtr ! 

Here are the updated changes @jarednova :
- Removing GT America and using default Helvetica
- On the sidebar nav, adding an underline to your current page
image

- Globally change text and code link styling to include a green underline
image
image
image

**Comment by jarednova** [comment]
Looking much closer! I think everything you flagged in the last pass @gchtr has been taken care of. There's one more area I'd like to spend a little more time. These tables with the method/property type and definitions:
Screen Shot 2020-11-13 at 11 25 55 AM

I was looking to apply some surrounding spans for `<span class="property">$language</span>` so that these can be style separately from the regular text. @gchtr does that markup/control reside within this repo? or in Teak?

**Review by gchtr — APPROVED** [review]
This is looking good now! I have only some changes in https://github.com/timber/docs/pull/30, but then this is good to go.

@jarednova I added some spans and classes to the Teak markup in the [1.0.3 release](https://github.com/timber/teak/compare/1.0.2...1.0.3). You can get it with `composer update timber/teak` in the docs repo. Will that work?

**Comment by jarednova** [comment]
@gchtr I think so! I applied a few last color/style tweaks. The last fix depends on a PR I just submitted to Teak here:

https://github.com/timber/teak/pull/1

... this will allow proper lines breaks in the MD files which trigger the correct markup from 11ty. Could you give this one last look before we merge? I want to make sure the changes I made in 1d703baf57396b8433d5da454599be25d95648e4 work (I needed to include `typeface-source-sans-pro` in order to run/build — was this correct?)

**Comment by gchtr** [comment]
> The last fix depends on a PR I just submitted to Teak here:
> 
> [timber/teak#1](https://github.com/timber/teak/pull/1)
> 
> ... this will allow proper lines breaks in the MD files which trigger the correct markup from 11ty.

Should be fixed now.

> Could you give this one last look before we merge?

I had a look and to me, all is fine now 👍

> I needed to include `typeface-source-sans-pro` in order to run/build — was this correct?)

I forgot to remove an entry in **webpack.mix.js**, that’s why that error appeared. Fixed in [`cbb056a` (#23)](https://github.com/timber/docs/pull/23/commits/cbb056af8e7dbe7d2aca9ea7c4ff0c20396b599f).

@jarednova I think you can go ahead and merge this.

**Comment by jarednova** [comment]
¡¡¡¡MOST EXCELLENT!!!!

**Pull request merged**